content/blog/2021-05-04-prometheus-conformance-remote-write-compliance.md

```diff
@@ -0,0 +1,134 @@
+    --- FAIL: TestRemoteWrite/telegraf/HonorLabels (14.61s)
+    --- FAIL: TestRemoteWrite/telegraf/Invalid (14.61s)
+    --- FAIL: TestRemoteWrite/telegraf/RepeatedLabels (14.61s)
+    --- FAIL: TestRemoteWrite/telegraf/Staleness (14.59s)
+    --- FAIL: TestRemoteWrite/telegraf/Up (14.60s)
+    --- PASS: TestRemoteWrite/telegraf/Counter (14.61s)
+    --- PASS: TestRemoteWrite/telegraf/Gauge (14.60s)
+    --- PASS: TestRemoteWrite/telegraf/Headers (14.61s)
+    --- PASS: TestRemoteWrite/telegraf/Histogram (14.61s)
+    --- PASS: TestRemoteWrite/telegraf/InstanceLabel (14.61s)
+    --- PASS: TestRemoteWrite/telegraf/JobLabel (14.61s)
+    --- PASS: TestRemoteWrite/telegraf/NameLabel (14.60s)
+    --- PASS: TestRemoteWrite/telegraf/Ordering (14.61s)
+    --- PASS: TestRemoteWrite/telegraf/SortedLabels (14.61s)
+    --- PASS: TestRemoteWrite/telegraf/Summary (14.60s)
+    --- PASS: TestRemoteWrite/telegraf/Timestamp (14.61s)
+--- FAIL: TestRemoteWrite/vector (0.01s)
+    --- FAIL: TestRemoteWrite/vector/Counter (10.02s)
+    --- FAIL: TestRemoteWrite/vector/EmptyLabels (10.01s)
+    --- FAIL: TestRemoteWrite/vector/Headers (10.02s)
+    --- FAIL: TestRemoteWrite/vector/HonorLabels (10.02s)
+    --- FAIL: TestRemoteWrite/vector/InstanceLabel (10.02s)
+    --- FAIL: TestRemoteWrite/vector/Invalid (10.02s)
+    --- FAIL: TestRemoteWrite/vector/JobLabel (10.01s)
+    --- FAIL: TestRemoteWrite/vector/Ordering (13.01s)
+    --- FAIL: TestRemoteWrite/vector/RepeatedLabels (10.02s)
+    --- FAIL: TestRemoteWrite/vector/Staleness (10.02s)
+    --- FAIL: TestRemoteWrite/vector/Up (10.02s)
+    --- PASS: TestRemoteWrite/vector/Gauge (10.02s)
+    --- PASS: TestRemoteWrite/vector/Histogram (10.02s)
+    --- PASS: TestRemoteWrite/vector/NameLabel (10.02s)
+    --- PASS: TestRemoteWrite/vector/SortedLabels (10.02s)
+    --- PASS: TestRemoteWrite/vector/Summary (10.02s)
+    --- PASS: TestRemoteWrite/vector/Timestamp (10.02s)
+--- FAIL: TestRemoteWrite/vmagent (0.01s)
+    --- FAIL: TestRemoteWrite/vmagent/Invalid (20.66s)
+    --- FAIL: TestRemoteWrite/vmagent/Ordering (22.05s)
+    --- FAIL: TestRemoteWrite/vmagent/RepeatedLabels (20.67s)
+    --- FAIL: TestRemoteWrite/vmagent/Staleness (20.67s)
+    --- PASS: TestRemoteWrite/vmagent/Counter (20.67s)
+    --- PASS: TestRemoteWrite/vmagent/EmptyLabels (20.64s)
+    --- PASS: TestRemoteWrite/vmagent/Gauge (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/Headers (20.64s)
+    --- PASS: TestRemoteWrite/vmagent/Histogram (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/HonorLabels (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/InstanceLabel (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/JobLabel (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/NameLabel (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/SortedLabels (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/Summary (20.66s)
+    --- PASS: TestRemoteWrite/vmagent/Timestamp (20.67s)
+    --- PASS: TestRemoteWrite/vmagent/Up (20.66s)
+````
```

Comment:
Add at least a one-sentence closer to this after the code block? Like how this is awesome, and stay tuned for further updates, or something.

Reply:
doen